src/databricks/labs/ucx/cli.py

```diff
@@ -260,5 +263,51 @@ def save_azure_storage_accounts(w: WorkspaceClient, subscription_id: str):
     azure_resource_permissions.save_spn_permissions()
 
 
+@ucx.command
+def save_aws_iam_profiles(w: WorkspaceClient, aws_profile: str | None = None):
+    """identifies all Instance Profiles and map their access to S3 buckets.
+    Requires a working setup of AWS CLI.
+    https://aws.amazon.com/cli/
+    The command saves a CSV to the UCX installation folder with the mapping.
+
+    The user has to be authenticated with AWS and the have the permissions to browse the resources and iam services.
+    More information can be found here:
+    https://docs.aws.amazon.com/IAM/latest/UserGuide/access_permissions-required.html
+    """
+    if not shutil.which("aws"):
+        logger.error(
+            "Couldn't find AWS CLI in path."
+            "Please obtain and install the CLI from https://aws.amazon.com/cli/"
+        )
+    if not aws_profile:
+        aws_profile = os.getenv("AWS_DEFAULT_PROFILE")
```

Comment:
If no config profile found - list all available profiles and let user pick. Use tui.Prompts from blueprint

Reply:
The list is too long (in our case).

Reply:
I'm not sure how to handle it.

Reply:
```python
aws_profile = prompt.question("AWS profile", default=os.environ.get("AWS_DEFAULT_PROFILE", aws_profile))
```